### PR TITLE
Fix #6594: Add 1.46 translations.

### LINKED ISSUE #6594: Add 1.46 translations

### PR DESCRIPTION
**Pull request by iccub**
## Summary of Changes

This pull request fixes #6594 

## Submitter Checklist:

- [x] *Unit Tests* are updated to cover new or changed functionality
- [x] User-facing strings use `NSLocalizableString()`
- [x] New or updated UI has been tested across:
  - [x] Light & dark mode
  - [x] Different size classes (iPhone, landscape, iPad)
  - [x] Different dynamic type sizes

## Test Plan:

Verify that P3A onboarding screen is translated.

## Reviewer Checklist:

- [x] Issues include necessary QA labels:
  - `QA/(Yes|No)`
  - `bug` / `enhancement`
- [x] Necessary [security reviews](https://github.com/brave/security/issues/new/choose) have taken place.
- [x] Adequate unit test coverage exists to prevent regressions.
- [x] Adequate test plan exists for QA to validate (if applicable).
- [x] Issue and pull request is assigned to a milestone (should happen at merge time).

### CODE REVIEW

**Review by soner-yuksel — APPROVED** [review]
Translations seems good for TR 🎉

File: Sources/BraveShared/Resources/zh.lproj/BraveShared.strings [hunk 1]
```diff
@@ -1303,6 +1303,12 @@
 /* The title of the view showing no synced sessions */
 "opentabs.noSyncSessionPlaceHolderViewTitle" = "此空间显示在其他设备中打开的 Brave 选项卡";
 
+/* The title for the action to hide open session with all tabs */
+"opentabs.openSessionHideAllActionTitle" = "暫時隱藏";
```
Comment (nuo-xu) [inline Sources/BraveShared/Resources/zh.lproj/BraveShared.strings]: "暫時隱藏" -> "暂时隐藏"

File: Sources/BraveShared/Resources/zh.lproj/BraveShared.strings [hunk 2]
```diff
@@ -3202,68 +3205,80 @@
...
+"today.availableRSSFeedsHeaderTitle" = "可用的 RSS 摘要";
 
 /* The name of the feature */
 "today.braveToday" = "Brave 新闻";
 
+/* Shown below 'Channels' explaining what will be shown when tapping on it. */
+"today.channelsButtonSubtitle" = "关注您感兴趣的主题";
+
+/* A button title that when tapped shows users a list of channels/topics that they can follow */
+"today.channelsButtonTitle" = "頻道";
```
Comment (nuo-xu) [inline Sources/BraveShared/Resources/zh.lproj/BraveShared.strings]: "頻道" -> "频道"

File: Sources/BraveShared/Resources/zh.lproj/BraveShared.strings [hunk 3]
```diff
@@ -3202,68 +3205,80 @@
...
+
+/* A header title shown over a list of channels (i.e. Cars, Business, Fashion, etc.) */
+"today.channelsHeaderTitle" = "頻道";
```
Comment (nuo-xu) [inline Sources/BraveShared/Resources/zh.lproj/BraveShared.strings]: "頻道" -> "频道"

File: Sources/BraveShared/Resources/zh.lproj/BraveShared.strings [hunk 4]
```diff
@@ -3202,68 +3205,80 @@
...
 "today.errorGeneralBody" = "Brave News 出现问题。请重试。";
 
 /* No comment provided by engineer. */
 "today.errorGeneralTitle" = "哎呀...";
 
+/* Shown below 'Following' explaining that they can manage their list of followed sources by tapping on it */
+"today.followingButtonSubtitle" = "管理您的摘要来源";
+
+/* A button title that when tapped shows users a list of sources & channels they are currently following */
+"today.followingButtonTitle" = "關注中";
```
Comment (nuo-xu) [inline Sources/BraveShared/Resources/zh.lproj/BraveShared.strings]: "關注中" -> "关注中"

File: Sources/BraveShared/Resources/zh.lproj/BraveShared.strings [hunk 5]
```diff
@@ -3202,68 +3205,80 @@
...
+
+/* A title at the top of the screen that shows a list of the news content the user is following */
+"today.followingTitle" = "關注中";
```
Comment (nuo-xu) [inline Sources/BraveShared/Resources/zh.lproj/BraveShared.strings]: "關注中" -> "关注中"

File: Sources/BraveShared/Resources/zh.lproj/BraveShared.strings [hunk 6]
```diff
@@ -3202,68 +3205,80 @@
...
+/* Shown above a button that turns on the \"Brave News\" product in the app. by sources & feeds they mean websites & blogs. */
+"today.introCardBody" = "通过单一摘要关注您喜爱的来源。只要在 Brave 中开启分页、向下滑动，一切搞定！\nBrave 新闻支援匿名投放私人广告。";
 
-/* No comment provided by engineer. */
-"today.introCardNewTextBody" = "使用我们的私有 CDN 交付的数百个主要内容源来定制您的订阅源，添加您最喜欢的出版商的 RSS 订阅源，制作您专属的订阅。";
-
-/* No comment provided by engineer. */
-"today.introCardTitle" = "今日热门故事，专属于您的私人提要。";
+/* Shown above a button that turns on the \"Brave News\" product in the app. */
+"today.introCardTitle" = "開啟 Brave 新聞，絕不錯過每一條消息";
```
Comment (nuo-xu) [inline Sources/BraveShared/Resources/zh.lproj/BraveShared.strings]: "開啟 Brave 新聞，絕不錯過每一條消息" -> "开启 Brave 新闻，绝不错过每一条消息"

**Review by nuo-xu — APPROVED** [review]

**Pull request merged**